set upstream ABI version as 0
Author: Mo Zhou
Forward: not-needed
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -255,6 +255,7 @@
   ADD_LIBRARY(cpuinfo ${CPUINFO_SRCS})
 ELSEIF(CPUINFO_LIBRARY_TYPE STREQUAL "shared")
   ADD_LIBRARY(cpuinfo SHARED ${CPUINFO_SRCS})
+  set_target_properties(cpuinfo PROPERTIES SOVERSION 0)
 ELSEIF(CPUINFO_LIBRARY_TYPE STREQUAL "static")
   ADD_LIBRARY(cpuinfo STATIC ${CPUINFO_SRCS})
 ELSE()
